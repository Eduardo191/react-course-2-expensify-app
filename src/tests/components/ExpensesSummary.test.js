import React from 'react';
import { ExpensesSummary } from '../../components/ExpensesSummary';
import { shallow } from 'enzyme';

test('should render ExpensesSummary component with one expense', () => {
    const wrapper = shallow(<ExpensesSummary expenseCount={1} expensesTotal={235} />);
    expect(wrapper).toMatchSnapshot();
});

test('should render ExpensesSummary component with multiple expenses', () => {
    const wrapper = shallow(<ExpensesSummary expenseCount={23} expensesTotal={2351414141} />);
    expect(wrapper).toMatchSnapshot();
});