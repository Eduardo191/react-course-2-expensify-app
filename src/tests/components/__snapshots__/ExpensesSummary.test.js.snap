// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render ExpensesSummary component with multiple expenses 1`] = `
<div>
  <h1>
    Viewing 
    23
     
    expenses
     tottaling 
    $23,514,141.41
  </h1>
</div>
`;

exports[`should render ExpensesSummary component with one expense 1`] = `
<div>
  <h1>
    Viewing 
    1
     
    expense
     tottaling 
    $2.35
  </h1>
</div>
`;
